compat/vcbuild/vcpkg_install.bat

```diff
@@ -36,6 +36,13 @@ REM ================================================================
 
 	dir vcpkg\vcpkg.exe >nul 2>nul && GOTO :install_libraries
 
+	git.exe version 2>nul
+	IF ERRORLEVEL 1 (
+	echo "***"
+	echo "Git not found. Please adjust you CMD path or Git install option."
```

Comment:
Small typo:

s/you/your/

Reply:
Fixed in 504b50d93d4f9665f00834f0c031967e89298787